### DISCUSSION TITLE
愚问

### DISCUSSION THREAD

**Discussion by nick-cjyx9** [ouuan/ouuan#36] [Q&A (中文)] [answered]
- 大佬有没有遇到过状态不佳，注意力涣散的时候呢，如果有是如何恢复的
- 怎么看待非oier学习编程，或者说您高中的算法竞赛知识在现在有多大程度上帮助到您呢
- 您是如何学习whk的呢

🥺感激不尽，orz

**Comment by ouuan** [comment]
> 有没有遇到过状态不佳，注意力涣散的时候呢，如果有是如何恢复的

~~以前没有恢复，现在感觉贴贴可以一定程度上恢复（x~~

**Reply by nick-cjyx9** [reply]
~~xm贴贴~~

**Reply by ouuan** [reply]
~~虽然确实挺值得 xm 的（（~~，但这一个月我也好想知道如何恢复（（（

**Comment by ouuan** [comment]
> 怎么看待非oier学习编程，或者说您高中的算法竞赛知识在现在有多大程度上帮助到您呢

感觉 OI 后期学的那些东西目前用处不太大，前期学的基础的东西还是多少有点用的，中期 (?) 大概是能混过数据结构这门课（只不过邓公 DSA 实在毒瘤，比起 CS:APP 读完随便水，还是上的挺难受）。OI 用处最大的大概还是对码力的训练，而不务正业的部分其实我感觉是比整个 OI 都更有用的（，引用一句玩笑话就是 CS 水平和 Linux 熟悉程度正相关（。但要说的话其实有用的部分也不多（只不过没用的知识也总是能锻炼有用的底力），如果要实用一点来说其实是挺浪费时间的，但问题就在于一直学最有用的东西学不学的下去 & 知不知道应该怎么学，反正我是不知道应该怎么学（（

**Comment by ouuan** [comment, marked as answer]
> 您是如何学习whk的呢

很怪，但其实我整个高三都没多少记忆（，只记得作业一如既往地做不完，晚自习看了很多看天下，周末回家（也就一天半）一点都不学（作业都是周日晚上去学校看老师哪个查的最紧就 rush 哪个），经常被老师拷打（~~拆东墙补西墙，所有老师都认为我忙着学别的科目（~~

![2021 contribution graph](https://github.com/ouuan/ouuan/assets/30581822/d9775a40-1ea0-4029-b828-92baa412ce7a)

~~虽然但是其实最后考的也不是很高（（~~，数学比想像中低一点，英语虽然一直还不错但高考是那一整年唯一一次上 140，化学生物是赋分，感觉主要是物理考的比较简单，然后总分作为强基生就还行（

**Reply by nick-cjyx9** [reply]
虽然学不来，但是还是感觉好受了一点😂感谢！
